go.mod

```diff
@@ -43,10 +43,7 @@ require (
 	github.com/evanphx/json-patch v4.9.0+incompatible
```

Comment:
@dims you will appreciate this one :-)

Reply:
![image](https://user-images.githubusercontent.com/23304/110048812-90fe2e00-7d1e-11eb-91b3-00d19e1b89a9.png)

Reply:
this is a good example of something we should mine for ... large dependency trees only required because of a tiny number of files... in this case, dropping a single test let us drop several problematic dependencies, including a mongodb driver (!)